### ISSUE TITLE
Multiple Features Requested!!!

### ISSUE THREAD

**Issue by andrewdwallo** [protonemedia/laravel-splade#73]
Hello,
1. It would be great to instead be able use the web middleware instead of a custom one considering so many Laravel packages rely on it and not yours.
2. While installing the Breeze demo app in the documentation many of your references don't coincide with the blade files 
(especially routes and views in the breeze demo plugin).. none of them have Vue components that are used as examples. I actually dont see anything Vue related in it..
Example: Link component isn't used for any routes in the blade files...
3. It would be nice to also have a library of .vue components to be able to reference to for building the app...

I am not criticizing. I am just asking for a feature/enhancement. Thanks!

**Comment by andrewdwallo** [comment]
Alright perhaps I just need to do more research with your plugin... So how would I go about using the web middleware group instead of the splade middleware group?

**Comment by davitlee88** [comment]
just use it.

**Comment by MotionPhix** [comment]
> Alright perhaps I just need to do more research with your plugin... So how would I go about using the web middleware group instead of the splade middleware group?

There's no conflict between the two. From what I have observed, I think you came here very quickly to criticize before even trying laravel splade out. I bet you give it time to read the documentation, try it out and then come back. We will take it from there.

**Comment by andrewdwallo** [comment]
Yeah your right I was kinda not in the right mind at that time tbh, I had been up for a couple days lmao.. my fault and apologies.. I was actually gonna ask for a feature that would be cool though.. Idk if its possible right now but could I use this along with Livewire?

**Comment by pascalbaljet** [comment]
I have yet to test it, but I highly doubt Splade is compatible with Livewire. Splade uses Vue under the hood.

**Comment by oceceli** [comment]
> I have yet to test it, but I highly doubt Splade is compatible with Livewire. Splade uses Vue under the hood.

Hi pascal, 

I admire your efforts put into this project. Thank you for that.

I've tried to use Livewire along with Splade. Here's my experience:
I've placed Livewire directives into the root blade which @splade directives in it. It seems to work at first glance, but when I navigate other routes and come back to the Livewire component, it loses its "Livewire features".
Is it possible to workaround the livewire components in the 'splade' middleware? Maybe bypass somehow?
